### PR TITLE
Kindling the Firebase

### PR DESCRIPTION
**Pull request by AlexGilleran**
Added a start for firebase.

- Facebook button actually login through facebook 💯 
- Hitting one of the experience circles creates a new task entity in firebase and immediately navigates to `/tasks/:taskId`. I wasn't sure whether it's supposed to be a new task every time or there's only ever one task per aspiration? Can change it pretty easily anyway.
- Editing the reminder frequency changes `users/${uid}/tasks/${taskId}/reminderFrequency`, and inviting a friend puts a new email address in `/invites` although you can't see it because it doesn't actually show the invites you've sent out anywhere (right?)

This uses re-base to do all the firebase database stuff - CreateTask.js uses `base.syncState` to keep `this.state` in sync with the task object in firebase, so when you use `setState` it updates firebase as well.

### CODE REVIEW

**Comment by AlexGilleran** [comment]
Ooops just realised I converted a bunch of stuff to tabs by accident, will fix it...

**Comment by andiwinata** [comment]
you can do `npm run format` to fix all code styling issue?

**Comment by AlexGilleran** [comment]
Oh so I can, super rad 💯

**Comment by AlexGilleran** [comment]
Oh btw go to https://console.firebase.google.com/u/0/project/iamee-2/database/data to see what it puts in the database. Even updates in real time :)

File: src/containers/templates/CreateTask/CreateTask.js [hunk 1]
```diff
@@ -57,20 +116,34 @@ class CreateTask extends Component {
...
+            <option value="weekly">Weekly</option>
+            <option value="monthly">Monthly</option>
           </Input>
         </div>
 
         <div className="exp-button width-80 btn-blue">
-          <BaseButton title={'SET REMINDER'} />
+          <BaseButton
+            title={'SET REMINDER'}
+            onClick={this.setReminder.bind(this)}
```
Comment (andiwinata) [inline src/containers/templates/CreateTask/CreateTask.js]: you can just declare `setReminder` as class properties so you don't need to bind all the time,

```
setReminder = () => {}
```

**Review by andiwinata — APPROVED** [review]
look really cool hey! I'll try checking the db tomorrow

**Pull request merged**